dashboard/app/views/errors/not_found.html.haml

```diff
@@ -0,0 +1,11 @@
+.error-parent
+  .error-child
+    %img{src: '/shared/images/sad-bee-avatar.png'}
```

Comment:
```suggestion
    %img{src: '/shared/images/sad-bee-avatar.png', alt: I18n.t(:page_not_found_title)}
```

Reply:
this still needs to be resolved

Reply:
thanks!  I resolved but I had conflicts with circle.yml files that had been removed and forgot to git stash pop the changes I made.